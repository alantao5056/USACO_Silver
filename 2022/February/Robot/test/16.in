40
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
578850532 209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
-578850532 -209992939
