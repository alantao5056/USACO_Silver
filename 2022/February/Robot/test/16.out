20
0
3800
0
216600
0
5523300
0
75116880
0
600935040
0
3004675200
0
9765194400
0
21157921200
0
31031617760
0
31031617760
0
21157921200
0
9765194400
0
3004675200
0
600935040
0
75116880
0
5523300
0
216600
0
3800
0
20
0
